contracts/TokenHub.sol

```diff
@@ -167,7 +167,7 @@ contract TokenHub is ITokenHub, System, IParamSubscriber, IApplication, ISystemR
     return actualAmount;
   }
 
-  function claimRewardsforFinality(address payable, uint256) onlyInit onlyRelayerIncentivize external override returns(uint256) {
+  function claimLargeRewards(address payable, uint256) onlyInit onlyRelayerIncentivize external override returns(uint256) {
     revert("CLAIM_REWARDS_FOR_FINALITY_NOT_ALLOWED");
```

Comment:
`CLAIM_REWARDS_FOR_FINALITY_NOT_ALLOWED`?

Reply:
After review and analysis by merge confirmation as completed